controllers/comments.js

```diff
@@ -0,0 +1,19 @@
+var Post = require('../models/comment');
+
+var CommentsController = {
+  Create: function(req, res) {
+    const comment = new Post({
+      comment: req.body.comment,
+      user_name: req.session.user.email, 
+      user_id: req.session.user.email._id,
+      post_id: req.body.post_id,
+      });
+    comment.save(function(err) {
+      if (err) { throw err; }
+
+      res.status(201).redirect('/posts');
+    });
+  }
+};
+
+module.exports = CommentsController;
```

Comment:
We haven't been strict around this so far (and eslint seems to ignore it), but we probably should try to have the final new line.